src/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -231,6 +231,7 @@ public sealed partial class MethodBuilder : System.Reflection.MethodInfo
         public bool InitLocals { get { throw null; } set { } }
         public override bool IsGenericMethod { get { throw null; } }
         public override bool IsGenericMethodDefinition { get { throw null; } }
+        public override bool IsConstructedGenericMethod { get { throw null; } }
```

Comment:
You either need to implement this one in the src project (in CoreCLR I assume), or remove the ref. @AtsushiKan @DnlHarvey will be able to provide guidance ...